packages/frontend/src/components/NavBar/GlobalSearch/SearchIcon.tsx

```diff
@@ -42,3 +49,56 @@ export const SearchIcon: FC<SearchIconProps> = ({ searchItem }) => {
             return <IconBox icon={IconAppWindow} color="gray.7" />;
     }
 };
+
+export const SearchIconWithIndicator: FC<{
+    searchResult: SearchItem;
+    projectUuid: string;
+    canUserManageValidation: boolean;
+}> = ({ searchResult, projectUuid, canUserManageValidation }) =>
+    searchResult.item && 'validationErrors' in searchResult.item ? (
```

Comment:
nit (can ignore): I personally find it hard to read ternary operator like this.
what I usually do is have an explicit "guard"

```
if (searchResult.item && 'validationErrors' in searchResult.item) return null

return component stuff

Reply:
Cool, yeah, I can see why but since this is a short condition I'll keep it as is 👍